### DISCUSSION TITLE
Stop Words in character files?

### DISCUSSION THREAD

**Discussion by OWKenobi** [oobabooga/text-generation-webui#793] [Ideas]
Some fine-tuned models like Vicuna have a tendency to fall out of role by introducing a higher level of abstraction. They do so by starting a new line with "### Assistant:" even though you are in a role play where for example User Llama talks with A.I. Penguin.

Now, to filter out this behavior, would it be beneficial to be able to introduce stop words in the character files? I was thinking about this field:

"stop_words": ["### Assistant:","### Human:"]

Whenever the model produces one of these stop_words, it will end output right before it.

**Comment by lwy123177** [comment]
Try adding
```
.*vicuna:
  custom_stopping_strings: '"### Assistant", "### Human"'
```
to
 `models/config.yaml`
